src/cs/Ssh.Tcp/Services/PortForwardingService.cs

```diff
@@ -594,7 +594,7 @@ public async Task WaitForForwardedPortAsync(
 		}
 	}
 
-	protected override async Task OnSessionRequestAsync(
+	protected internal override async Task OnSessionRequestAsync(
```

Comment:
I don't think this is necessary? Tests are not calling `OnSessionRequestAsync()`.

Same for line 799.